js/models/itemModel.js

```diff
@@ -15,7 +15,7 @@ export default class ItemModel extends LockingModel {
   }
 
   toggleActive(isActive = !this.get('_isActive')) {
-    this.set('_isActive', isActive);
+    this.set('_isActive', !!isActive);
```

Comment:
```suggestion
    this.set('_isActive', Boolean(isActive));
```

Reply:
Why is this needed? Which bit of code is trying to set it to a non-boolean value?

Reply:
`restoreUserAnswers()` within the `itemsQuestionModel.js` file, we are seeing it is passing a binary value and not boolean.

Reply:
https://github.com/adaptlearning/adapt-contrib-core/blob/de6fcd9c58ad7ed0d793920b048fcb213bdcc980/js/models/itemsQuestionModel.js#L47

Could you please wrap that line in a Boolean() call? As it's the offending cause.

Reply:
Yeah done and understood. Thanks Oliver

Reply:
I'm happy for you to leave your original bit in also in toggleActive.